release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -113,6 +113,11 @@ For more information, see xref:../security/audit-log-policy-config.adoc#configur
 [id="ocp-4-9-networking"]
 === Networking
 
+[id="ocp-4-9-ptp-fast-event-notifications"]
+==== Monitoring PTP fast events with the PTP fast event notification framework
+
+Fast event notifications for PTP events are now available for bare-metal clusters. The PTP Operator generates event notifications for every configured PTP-capable network interface. Events are made available through a REST API for applications running on the same node. Fast event notifications are transported by an Advanced Message Queuing Protocol (AMQP) message bus provided by the AMQ Interconnect Operator. For more information, see xref:../networking/configuring-ptp.adoc#cnf-about-ptp-and-clock-synchronization_configuring-ptp[About PTP and clock synchronization error events].
```

Comment:
Telco docs appear to be the only place to use "bare-metal" instead of "bare metal."

Reply:
"Bare-metal" is the adjectival form, "bare metal" is the noun form. https://redhat-documentation.github.io/supplementary-style-guide/#baremetal

https://redhat-documentation.github.io/supplementary-style-guide/#bare-metal